src/test/scala/cromwell/engine/WorkflowManagerActorSpec.scala

```diff
@@ -115,4 +115,17 @@ class WorkflowManagerActorSpec extends CromwellTestkitSpec("WorkflowManagerActor
       }
     }
   }
+
+  it should "error when running a workflowless WDL" {
+    DataAccess.withDataAccess { dataAccess =>
```

Comment:
I better not see `mcovarr` on the blame mode of the other tests in this class when I see whom I copied it from ;)